src/components/AssetSelection/components/AssetChainDropdown/AssetChainDropdown.tsx

```diff
@@ -77,11 +87,11 @@ export const AssetChainDropdown: React.FC<AssetChainDropdownProps> = memo(
       [onChangeAsset],
     )
 
-    const isDisabled = useMemo(() => {
-      return filteredRelatedAssetIds.length <= 1 || isLoading || isError
-    }, [filteredRelatedAssetIds, isError, isLoading])
+    const isButtonDisabled = useMemo(() => {
```

Comment:
Similar naming sanity here, making things clear re: making it clear this is the `isDisabled` directive of the button, not anything else